home.html

```diff
@@ -0,0 +1,165 @@
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+
+        <!-- desktop version  -->
+        <div class="list_navs">
+            <ul>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#project">Project</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+        
+    </nav>
+     
+    <!-- hero section  -->
+    <div class="head_content">
+        <h3>"Hello Investors!"</h3>
+        <h2>Find that <br>elusive Unicorns</h2>
+    </div>
+    <div class="header_quote">
+        <p><span><strong>SANO</strong></span> will enable you to discover the newest startups to invest in. It is used by millions of people Worldwide with the best customer satisfaction.</p>
+    </div>
+    <div class="invite_date">
+        <h2><i>COME SEE US IN THE FREE WEBINER</i></h2>
+        <H3>2022:26:11(SAT) ~ 27(SUN)</H3>
+        <p>@International startups Meet, Nigeria, Abuja.</p>
+    </div>
+</div>
+
+<!-- program section  -->
+<div class="programs" id="program">
+    <div class="program">
+        <h3>MAIN PROGRAM</h3>
+        <img src="./images/line2.png" alt="line" width="50px">
+    </div>
+    <div class="programmer">
+        <div class="program_list">
+            <img src="./images/conference.png" alt="lecture">
+            <h3>lecture</h3>
+            <p>Listen to speakers from different part of the world.</p>
+        </div>
+        <div class="program_list">
+            <img src="./images/exhibition (1).png" alt="exhibition">
+            <h3>Showcase</h3>
+            <p>watch people from various countries Showcase their startups</p>
+        </div>
+        <div class="program_list">
+            <img src="./images/chat-bubble.png" alt="forum">
+            <h3>Forum</h3>
+            <p>Have time to share your thoughts and experiences.</p>
+        </div>
+        <div class="program_list">
+            <img src="./images/workshop (1).png" alt="workshop">
+            <h3>workshop</h3>
+            <p>Have a real working experience with colleages</p>
+        </div>
+        <div class="program_list">
+            <img src="./images/networking.png" alt="ignite">
+            <h3>Network</h3>
+            <p>Get oppurtunities to Network with other people.</p>
+        </div>
+    </div>
+    <div class="see_more">
+        <p><a href="#">SEE THE WHOLE PROGRAM</a></p>
+    </div>
+    <button type="submit" class="btn_submit">JOIN THE WEBINER NOW!</button>
+</div>
+<!-- feature speaker section -->
+<section id="feature">
+<div class="feature">
+    <h2>Featured speakers</h2>
+    <img src="./images/line2.png" alt="line" width="50px">
```

Comment:
- [ ] Make sure that the home page looks exactly like the provided design. kindly consider changing the background of the speakers sections to a white background so your page looks exactly as the provided design

|Your Page|CC design|
|-|-|
|![Screenshot_20221123_201636](https://user-images.githubusercontent.com/25227022/203629400-7b49f3c8-2786-4e31-b726-59da030d4aef.png)|![Screenshot_20221123_201717](https://user-images.githubusercontent.com/25227022/203629425-9526ffbc-3e49-4d68-8e6a-0c14c14732de.png)|